### PR TITLE
[video] resolve settings button visibility

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/plugin-video/web/src/getToolbarButtons.tsx [hunk 1]
```diff
@@ -59,6 +59,12 @@ export const getToolbarButtons = (config, services): ToolbarButton[] => {
         id: videoModals.settings,
         Component: VideoSettingsModal,
       },
+      attributes: {
+        visible: {
+          id: 'IS_VIDEO_SETTINGS_BUTTON_VISIBLE',
+          resolve: () => !!config.shouldShowSpoiler,
```
Comment (danarish4) [inline packages/plugin-video/web/src/getToolbarButtons.tsx:65]:
```suggestion
          resolve: () => {
          const spoilerPlugin = services.plugins.asArray().find(plugins => plugin.type === spoiler)
          return !spoilerPlugin.getConfig().supportedPlugins || spoilerPlugin.getConfig().supportedPlugins.includes(video) 
          },
```

**Review by danarish4 — CHANGES_REQUESTED** [review]
You can use the `EditorPlugins` entity for checking if a spoiler plugin exists.
you can revert the changes in `helpers-config` file

File: packages/plugin-video/web/src/getToolbarButtons.tsx (hunk 1 above)
Comment (tamrat-bay) [reply packages/plugin-video/web/src/getToolbarButtons.tsx:65]: we don't have plugins in services
services.plugins === undefined

Comment (danarish4) [reply packages/plugin-video/web/src/getToolbarButtons.tsx]: It should be there
https://github.com/wix/ricos/blob/master/packages/ricos-types/web/src/services.ts#L13

**Pull request merged**